line_follower_control:
  ros__parameters:
    wheel_diameter: 0.14
    wheel_base: 0.5
    max_linear_speed: 1.11  # 4 km/h
    rated_motor_rpm: 2750.0
    default_linear_speed: 0.2
    camera_fov_deg: 60.0
    image_width: 960
    control_rate: 50.0
    port: "/dev/ttyUSB0"
    baudrate: 115200
    kp: 0.8
    ki: 0.02
    kd: 0.05
